etc/sqre/config.yaml

```diff
@@ -58,13 +58,9 @@ wget:
     tag: latest
 squash:
   url: https://squash-restful-api.lsst.codes/
-layercake:
-  packer:
-    github_repo: lsst-sqre/packer-layercake
-    git_ref: master
-    dir: ''
+scipipe_base:
```

Comment:
Should note somewhere that this kills the build_layercake (with sqre/infra/build_layercake pipeline), build_newinstall_triggers, and infra_monthly jobs, perhaps even moving them into some "disabled" place?

Reply:
I'll create a Jira ticket for this work (it's currently down):

Title: Jenkins jobs to sunset build_layercake for build_docker_scipipe
- Create a build_docker_scipipe job for building docker-scipipe repo
- The layercake job should be disabled or relocated to somewhere else in jenkins
- The infra jobs and trigger job should be updated